zincware/MDSuite
# Mac M1 update.

Fix #236 

#### Summary of additions and changes

* Added documentation for the Mac silicon installation.
* Updated index.html to reflect this.

#### How to test this pull request

```sh
cd MDSuite/docs
make html
firefox index.html
```
Go and check out the installation docs.
